EntAppProject/src/main/resources/templates/JBLogin.html

```diff
@@ -1,45 +1,102 @@
 <!DOCTYPE html>

```

Comment:
This is probably the most important HTML page change that I did for this code review. This is the revamped Login page with bootstrap included. The output of this page is a huge improvement and the Stylesheets included help to provide new aesthetics to the application. Also, the app uses local files to support the page which means the team can make any adjustments to it, including colors and spacing.